www/content/workstation/install_workstation.md

```diff
@@ -122,6 +122,10 @@ macOS
     **Download** button.
 3.  Follow the steps to accept the license and install Chef Workstation.
 
+Alternately, install Chef Workstation using Homebrew:
+
+`brew cask install chef/chef/chef-workstation`
```

Comment:
```suggestion
`brew cask install chef-workstation`
```

Now that https://github.com/Homebrew/homebrew-cask/pull/81618 is merged we no longer need to use a custom Homebrew tap!